source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
Yes, then we must create `div` every time  `createNode` is invoked, or define ` let div` and every time check inside function, if `undefined` create. I'm not sure, what the best.